src/components/Sidebar.js

```diff
@@ -0,0 +1,73 @@
+import { DUMMY_CHARACTER } from "../constants/constants.js";
+import Component from "../core/Component.js";
+
+export default class Sideber extends Component{
+  template(){
+    const {documents, id} = this.props;
+    return `
+      ${this.getDirectoryTree(documents, +id)}
+      <button class="add-root">신규 문서 추가</button>
+    `
+  }
+
+  setEvent(){
+    const {onClickAdd, onClickDelete, onClickDocument} = this.props;
+    this.addEvent("click", ".add-root", () => {
+      onClickAdd(null);
+    })
+
+    this.addEvent("click", ".directory-entry", ({target}) => {
+      const $entry = target.closest(".directory-entry");
+      const {id} = $entry.dataset;
+      const {classList} = target;
+    
+      if(classList.contains("add-document")){
+        onClickAdd(id);
+      }else if(classList.contains("delete-document")){
+        onClickDelete(id);
+      }else if(classList.contains("selected-document-title")){
+        onClickDocument(id);
+      }else if(classList.contains("toggle-content")){
+        return;
+      }else{
+        onClickDocument(id);
+      }
+    })
+
+    this.addEvent("click", ".toggle-content", ({target}) => {
+      const $directoryEntry = target.closest(".directory-entry");
+      const {id} = $directoryEntry.dataset;
+      const $targetContent = document.querySelector(`#children-${id}`);
+      if($targetContent.style.display === "block" || $targetContent.style.display === ""){
+        $targetContent.style.display = "none";
+      }else{
+        $targetContent.style.display = "block";
+      }
+    })
+  }
+
+  getDirectoryTree(documents, selectedId){
+    if(!Array.isArray(documents) || documents.length === 0){
+      return ``;
```

Comment:
```suggestion
      return '';
```
사소하다면 사소할 수 있지만 백틱(`)을 사용하는 의미가 없는경우는 작은 따옴표(')를 쓰는게 좋을것 같습니다 👍